src/components/core/seo.js

```diff
@@ -11,16 +11,17 @@ const defaultSiteMetadata = {
   author: "@livechat",
 };
 
-function SEO({ desc, keywords, title }) {
+function SEO({ desc, keywords, title, subtitle, robots }) {
```

Comment:
You didn't add these props to PropTypes. Check the validity of passed props to avoid problems in the future.